commands/fetch.ts

```diff
@@ -80,12 +91,30 @@ exports.builder = yargs => {
     },
   });
 
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
-  addOverwriteOptions(yargs);
```

Comment:
Does makeYargsBuilder support `useOverwriteOptions` to replace this?

Reply:
No, I deleted this by mistake and have a fix pending local.  It's only used in two places, so I think calling it in the builder makes sense.